tests/test_page.py

```diff
@@ -339,3 +341,70 @@ def test_is_minor_or_minor_version():
     assert page.version == "2.0"
     assert page.is_major_version() is True
     assert page.is_minor_version() is False
+
+
+@pytest.mark.parametrize(
+    "page_versions, expected_order",
+    (
+        (["1.0", "1.1", "1.2", "2.0"], ["2.0", "1.2", "1.1", "1.0"]),
+        (["2.0", "1.2", "1.1", "1.0"], ["2.0", "1.2", "1.1", "1.0"]),
+    ),
+)
+def test_get_measure_page_versions_returns_pages_ordered_by_version(
+    db_session, page_service, stub_measure_page, page_versions, expected_order
+):
+    create_measure_page_versions(db_session, stub_measure_page, page_versions)
+
+    assert [
+        page.version
+        for page in page_service.get_measure_page_versions(
+            parent_guid=stub_measure_page.parent.guid, measure_uri="test-measure-page-2"
+        )
+    ] == expected_order
+
+
+@pytest.mark.parametrize(
+    "page_versions, expected_version", ((["1.0", "1.1", "1.2", "2.0"], "2.0"), (["2.0", "1.2", "1.1", "1.0"], "2.0"))
+)
+def test_get_latest_version_returns_latest_measure_page(
+    db_session, page_service, stub_measure_page, page_versions, expected_version
+):
+    create_measure_page_versions(db_session, stub_measure_page, page_versions)
+
+    assert (
+        page_service.get_latest_version(
+            topic_uri=stub_measure_page.parent.parent.uri,
+            subtopic_uri=stub_measure_page.parent.uri,
+            measure_uri="test-measure-page-2",
+        ).version
+        == expected_version
+    )
+
+
+@pytest.mark.parametrize(
+    "page_versions, expected_order", ((["1.0", "2.0"], ["1.0", "2.0", "1.0"]), (["2.0", "1.0"], ["1.0", "2.0", "1.0"]))
+)
+def test_get_pages_by_type_returns_pages_ordered_by_title_and_version(
+    db_session, page_service, stub_measure_page, page_versions, expected_order
+):
+    create_measure_page_versions(db_session, stub_measure_page, page_versions)
+
+    pages = page_service.get_pages_by_type("measure")
+    assert [page.title for page in pages] == ["Test Measure Page", "Test measure page 2", "Test measure page 2"]
```

Comment:
Maybe add a comment that the fist one comes from the `stub_measure_page` fixture and the others are new versions created for the test? Took me a little while to clock why you get three things back after creating two.

Reply:
Yep, done.